start

```diff
@@ -116,6 +123,10 @@ function process_args() {
 		exit 1
 	esac
 
+	if [ "$RANDOMIZE_NETWORK_PASSPHRASE" = true ]; then
+		NETWORK_PASSPHRASE+=$RANDOM
```

Comment:
❗ `$RANDOM` will return a random 16 bit integer. If `openssl` is installed in the container I'd use `openssl rand -hex 32` to generate a more secure random value.

Reply:
cool TIL!
https://github.com/stellar/docker-stellar-core-horizon/pull/276/commits/e28e8e783d363d8725dcf0a70f43bc71cbfa310b